crates/sui/Cargo.toml

```diff
@@ -53,10 +53,6 @@ workspace-hack = { path = "../workspace-hack"}
 multiaddr = "0.14.0"
 bcs = "0.1.3"
 
-[target.'cfg(not(target_env = "msvc"))'.dependencies]
-jemallocator = { version = "^0.5", features = ["profiling"] }
```

Comment:
This one we actually want to keep, and it's worth annotating it as ignorable by udeps.